src/views/callback.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <LoadingModal> Récupération de vos informations en cours… </LoadingModal>
+</template>
+
+<script>
+import axios from "axios"
+
+import LoadingModal from "@/components/loading-modal.vue"
+
+export default {
+  name: "Callback",
+  components: {
+    LoadingModal,
+  },
+  async mounted() {
+    const search = new URLSearchParams(document.location.search)
```

Comment:
C'est plus adéquat d'utiliser `this.$route` que `document.location` pour les composants vue (valable aussi pour les lignes 32 et 35)

Reply:
Non ça casse la redirection aussi, j'ai l'impression qu'il nous faut un vrai changement de page ici et pas un seulement dans vue

Reply:
Ok my bad !